src/server/admin/tags/controllers/list-tags.js

```diff
@@ -0,0 +1,23 @@
+import { serviceTags } from '~/src/server/admin/tags/helpers/service-tags.js'
+import { transformTagToRow } from '~/src/server/admin/tags/transformers/transform-tag-to-row.js'
+
+const listTagsListController = {
+  handler: (request, h) => {
+    const rows = Object.values(serviceTags).map(transformTagToRow)
+
+    return h.view('admin/tags/views/tags-list', {
+      pageTitle: 'Tags',
+      tableData: {
+        headers: [
+          { id: 'tag', text: 'Tag', width: '3' },
```

Comment:
FYI to be responsive - the widths should add up to a total of 100. I suspect this may look a little off on some screens?